### ISSUE TITLE
Implement Rewards Backup & Restore 2.0 

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#18101]
## Description

Implement Rewards Backup & Restore 2.0, which will allow user with unverified Rewards wallets to backup and restore their Brave Rewards wallet with vBAT in a privacy-preserving way.

**Comment by Saoiray** [comment]
Hey all. Sorry to bother but was wondering if any general update on this? 

With the huge amount of regions being unsupported by Gemini and Uphold lately, this means a significant amount of Users are stuck with BAT only in their browser. Should things go wrong, they lose it all. It seems like this feature should be more of a high priority as a result. 

@szilardszaloki @Miyayes ?

**Comment by Miyayes** [comment]
@Saoiray As you can see in the PR https://github.com/brave/brave-core/pull/13757, a tremendous amount of work went into this, and the feature was going through the code review process. However, some big changes will be coming to Brave Rewards in the near future that will make Backup & Restore less relevant. As a result, we decided in a meeting last week (after considerable deliberation) that Backup & Restore be put on pause. 

In the meantime, users facing a situation in which they have no custodial options and need to back up their current vBAT (for example, if they need to reformat) can manually copy-paste/back up their `/brave-browser` folder, at least on desktop. They can find a path to it in `brave://version`.

**Comment by Saoiray** [comment]
>As you can see in the PR https://github.com/brave/brave-core/pull/13757, a tremendous amount of work went into this, and the feature has been going through the code review process.

Yeah, I know y'all are always busy with things. I eventually need to learn to navigate through Github better. I usually just go through https://github.com/orgs/brave/projects/28#card-59568971 and then read summaries on cards. Guess I'll hope to be seeing/hearing what's going on in big changes here soon. 

Thanks for the response.

**Comment by UjCbFwtBayFM** [comment]
> However, some big changes will be coming to Brave Rewards in the near future that may make Backup & Restore less relevant.

Care to elaborate on these big changes?

**Comment by Pratham-Gulati** [comment]
please can you explain in simpler terms how to save it locally
